.github/workflows/patrol-publish.yaml

```diff
@@ -49,4 +52,7 @@ jobs:
           SLACK_COLOR: ${{ steps.slack_message.outputs.status }}
           SLACK_ICON: https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png
```

Comment:
Make `SLACK_CHANNEL` and `SLACK_ICON` repository variables

Reply:
`SLACK_CHANNEL` is `dev-testers` in `patrol_cli-publish` and `patrol-publish`

`SLACK_CHANNEL` is `patrol-builds` in all the integration test actions.

I'll add `SLACK_STATUS_CHANNEL` and `SLACK_RELEASES_CHANNEL`, sounds ok?